cockroachdb/values.yaml

```diff
@@ -261,6 +273,17 @@ ingress:
   # - hosts: [cockroachlabs.com]
   #   secretName: cockroachlabs-tls
 
+# CockroachDB's metrics service
+metrics:
```

Comment:
For any new values added to values.yaml, please [document them in the README](https://github.com/cockroachdb/helm-charts/tree/master/cockroachdb#configuration).